Teapot/Teapot+iOSMultipartDataForImage.swift

```diff
@@ -1,3 +1,4 @@
+#if os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
how was this not causing issues on the `macOS` target before?

Reply:
I think no one was using it 😭